tests/queries/0_stateless/02921_database_filesystem_path_check.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+function get_exception_message()
+{
+  $CLICKHOUSE_CLIENT --query "$1" |& grep -o 'Path must.*path'
```

Comment:
The test is failing because it matches 2 strings and not one.

In this case I'd recommend another test type, which is simpler:
* Use a .sql (no executable rights)
* Place the queries here + the expected error code.
* An empty reference file.

The test runner will execute the queries and will fail if the error code doesn't match, e.g:

`02921_database_filesystem_path_check.sql`

```
create database db_filesystem ENGINE=Filesystem('/etc'); -- { serverError BAD_ARGUMENTS }
create database db_filesystem ENGINE=Filesystem('../../../../../../../../etc'); -- { serverError BAD_ARGUMENTS }
```


BTW, you can check that the test passes by starting a CH server with your changes and running:
```
./tests/clickhouse-test 02921_database # Whatever is enough to match just the name of your test
```

Reply:
thanks @Algunenano ... 🤦  I did manually run the command in my mac and check output but the result turns out to be different to what ran on CI and greedy grep bite ... I will make use of sql check and try out the test locally ... just need to install jinja and iron out my janky dev environment...